### DISCUSSION TITLE
arg[0].indexOf is not a function

### DISCUSSION THREAD

**Discussion by demo0102** [dani-garcia/vaultwarden#2099] [Q&A]
### Subject of the issue

when I want to log in the web, it shows "arg[0].indexOf is not a function".
### Deployment environment

* vaultwarden version:1.23.0

* Install method:Docker image

* Clients used: web vault

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Expected behaviour

log in successfully.
### Actual behaviour

can't log in 
### Troubleshooting data

![image](https://user-images.githubusercontent.com/42965600/141671308-ca7684b5-ce30-4f2d-8781-ec787da82911.png)

**Comment by BlackDex** [comment]
You are probably not using HTTPS.
https://github.com/dani-garcia/vaultwarden/wiki/Enabling-HTTPS
https://github.com/dani-garcia/vaultwarden/wiki/Running-a-private-vaultwarden-instance-with-Let%27s-Encrypt-certs

**Reply by demo0102** [reply]
> You are probably not using HTTPS. https://github.com/dani-garcia/vaultwarden/wiki/Enabling-HTTPS https://github.com/dani-garcia/vaultwarden/wiki/Running-a-private-vaultwarden-instance-with-Let%27s-Encrypt-certs

I have used nginx in synology to enable https, but stiil not working.

**Reply by demo0102** [reply]
> You are probably not using HTTPS. https://github.com/dani-garcia/vaultwarden/wiki/Enabling-HTTPS https://github.com/dani-garcia/vaultwarden/wiki/Running-a-private-vaultwarden-instance-with-Let%27s-Encrypt-certs

Well, I found when I use chrome I get the message, but when I use other browser like safari, I can log in successfully.

**Reply by BlackDex** [reply]
Sometimes since extensions are breaking stuff. Maybe there is one you are using which breaks it.
